sycl/include/sycl/ext/oneapi/experimental/enqueue_functions.hpp

```diff
@@ -292,23 +296,10 @@ template <typename KernelName = sycl::detail::auto_name, int Dimensions,
           typename Properties, typename KernelType, typename... ReductionsT>
 void nd_launch(queue Q, launch_config<nd_range<Dimensions>, Properties> Config,
                const KernelType &KernelObj, ReductionsT &&...Reductions) {
-#ifdef __DPCPP_ENABLE_UNFINISHED_NO_CGH_SUBMIT
-  // TODO The handler-less path does not support reductions yet.
-  if constexpr (sizeof...(ReductionsT) == 0) {
-    ext::oneapi::experimental::detail::LaunchConfigAccess<nd_range<Dimensions>,
-                                                          Properties>
-        ConfigAccess(Config);
-    detail::submit_kernel_direct<KernelName>(
-        std::move(Q), ConfigAccess.getProperties(), ConfigAccess.getRange(),
-        KernelObj);
-  } else
-#endif
-  {
-    submit(std::move(Q), [&](handler &CGH) {
-      nd_launch<KernelName>(CGH, Config, KernelObj,
-                            std::forward<ReductionsT>(Reductions)...);
-    });
-  }
+  submit(std::move(Q), [&](handler &CGH) {
+    nd_launch<KernelName>(CGH, Config, KernelObj,
+                          std::forward<ReductionsT>(Reductions)...);
+  });
```

Comment:
why do we always use the handler path in this case? Are we planning to change it in the future?